README.md

```diff
@@ -61,20 +61,21 @@ You will need to have a native client application registered with Microsoft usin
 [Azure portal](https://portal.azure.com). 
 
 1. Getting to app registration
-    - Navigate to the [Azure portal](https://aad.portal.azure.com).  
-    - Click on ***Azure Active Directory*** > ***App Registrations***. 
+    - Navigate to the [Azure portal > App registrations](https://aad.portal.azure.com) to register your app. Sign in using a work or school account, or a personal Microsoft account. 
 
 2. Create the app
-    - Click ***New application registration***.  
-    - Enter an app name in the ***Name*** field. 
-    - In ***Application type***, select `Native`. 
-    - In ***Redirect URI***, enter `http://localhost`.  
+    - Select ***New registration***.  
+    - Enter an application name in the ***Name*** field. 
+    - Set ***Supported account types*** to ***Accounts in any organizational directory***. 
+    - Leave ***Redirect URI*** `http://localhost`.  
```

Comment:
Recommend the following for this:

- Set the ***Redirect URI***  field to `Public client` and add the value `http://localhost`.

Reply:
@danieldobalian updated the Redirect URI guidance - please review.